roles/docker/tasks/cleanup.yml

```diff
@@ -1,6 +1,14 @@
 ---
 # This play contains tasks for cleaning up docker
 
+- name: remove all containers
+  shell: unset DOCKER_HOST && docker rm -f -v $(docker ps -a -q)
```

Comment:
Will this delete swamr-master and swarm-agent container ? 
So one I run uninstaller, it will cleanup etcd persistent storage and delete all docker containers and delete docker engine as well ?